extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -237,8 +250,9 @@ define('ext.wikia.adEngine.adContext', [
 		context.opts.netzathleten = isEnabled('wgAdDriverNetzAthletenCountries');
 		context.opts.additionalVastSize = isEnabled('wgAdDriverAdditionalVastSizeCountries');
 
-		// Need to be placed always after all lABrador wgVars checks
-		context.opts.labradorDfp = geo.mapSamplingResults(instantGlobals.wgAdDriverLABradorDfpKeyvals);
+		if (context.targeting.skin === 'oasis') {
+			trackLabrador();
```

Comment:
I'm worried if it's not too early :( (e.g. for bill-the-lizard), also if it's good place (tracking in context). What do you think about:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/run/desktop.run.js#L106

Reply:
Done.